src/components/OrgLogoUpload.tsx

```diff
@@ -104,8 +105,8 @@ export const OrgLogoUpload = ({
                 background: '$white',
                 boxShadow: '$heavy',
                 borderRadius: '$3',
-                paddingLeft: '$2xl',
-                height: '$2xl',
+                paddingLeft: '64px',
```

Comment:
whats the story here

Reply:
This was for the org avatar upload.  I can do it with a calc, so I'll make that change.  

<img width="366" alt="Screen Shot 2022-09-16 at 3 55 56 PM" src="https://user-images.githubusercontent.com/100873710/190828396-9cb9cfd6-5cbf-443b-8330-171f25b28537.png">